src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -257,6 +258,11 @@ public class Feature
         /// </summary>
         public static Feature DocumentValidation => __documentValidation;
 
+        /// <summary>
+        /// Gets the directConnection feature.
+        /// </summary>
+        public static Feature DirectConnection => __directConnection;
```

Comment:
Rename to `DirectConnectionSetting`

It's the **Setting** that is a new feature... direct connections themselves were already supported.